src/stdlib_hashmap_wrappers.f90

```diff
@@ -277,6 +279,21 @@ subroutine get_int8_key( key, value )
     end subroutine get_int8_key
 
 
+    subroutine get_int32_key( key, value )
+!! Version: Experimental
+!!
+!! Gets the contents of the key as an INTEGER(INT8) vector
```

Comment:
Should this comment have `INTEGER(INT32)` rather than `INTEGER(INT8)` ?

Reply:
This should be updated per latest revision